### PR TITLE
Add db_instance terraform module

### PR DESCRIPTION
**Pull request by Nmishin**

### CODE REVIEW

File: db_instance/main.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+resource "mcs_db_instance" "main" {
+  name = var.db_instance_name
+
+  keypair = var.db_instance_keypair
+
+  datastore {
+    type    = "mysql"
+    version = "5.7"
```
Comment (satamas) [inline db_instance/main.tf:8]: 5.7 is a quite old version of MySql, why do we use it?

File: db_instance/versions.tf [hunk 2]
```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_version = "1.0.4"
+  required_providers {
+    mcs = {
+      source = "MailRuCloudSolutions/mcs"
```
Comment (satamas) [inline db_instance/versions.tf:5]: Can you explain how this custom provider works?

File: db_instance/main.tf (hunk 1 above)
Comment (Nmishin) [reply db_instance/main.tf]: add it as separate variable, by default v 8.0

File: db_instance/versions.tf (hunk 2 above)
Comment (Nmishin) [reply db_instance/versions.tf:5]: this is custom provider written by Vk cloud team, it convert HCL (terraform) language to the API calls understandable to custom's Vk cloud solutions

Comment (Nmishin) [reply db_instance/versions.tf:5]: by default providers downloads from the terraform registry:
https://registry.terraform.io/providers/MailRuCloudSolutions/mcs/latest
I will add that link to the Readme

**Pull request merged**